schema.graphql

```diff
@@ -312,4 +312,4 @@ input UpdateExportLocationInput {
   exportDatetime: ZonedDateTime
 }
 
-scalar ZonedDateTime
\ No newline at end of file
+scalar ZonedDateTime
```

Comment:
This is the one file where we can't have a line break at the end, as it causes the diffing to get messed up, if I remember correctly.